Encryption.md

```diff
@@ -19,15 +19,15 @@
 
 # Parquet Modular Encryption
 
-Parquet files, containing sensitive information, can be protected by the modular
-encryption mechanism, that encrypts and authenticates the file data and metadata - 
+Parquet files containing sensitive information can be protected by the modular
+encryption mechanism that encrypts and authenticates the file data and metadata - 
 while allowing for a regular Parquet functionality (columnar projection, 
-predicate pushdown, encoding and compression). The mechanism also enables column access 
+predicate pushdown, encoding and compression). This mechanism also enables column access 
```

Comment:
I don't think this document should make the claim that this "enables column access control". Using different encryption keys can be used in a scheme that provides some form of access control, but in itself it doesn't provide what we commonly think of as "access control" because there is no way to revoke that access.

This document should be clear about what the feature provides -- the ability to encrypt columns with different keys or not encrypt a column -- and should not make claims about security guarantees that can be partilly built with these features.